cluster/gce/gci/configure-helper.sh

```diff
@@ -2921,6 +2921,8 @@ function main() {
   if [[ "${container_runtime}" == "docker" ]]; then
     assemble-docker-flags
   elif [[ "${container_runtime}" == "containerd" ]]; then
+    # stop docker if it is present as we want to use just containerd
+    systemctl stop docker || echo "unable to stop docker"
```

Comment:
If docker cannot be stopped, would there be any downsides later on when using containerd?

Reply:
@karan no downsides. this is just to prevent accidental use of docker.

Reply:
If stopping docker fails, then we might not actually be able to test containerd itself right?

Reply:
this was not about stopping docker, there are some images where docker is not present, this was to tolerate those. (we have some ubuntu based ci jobs where we install versions of containerd as required and no docker)

containerd and docker are separate services since 18.09.x, majority of our images / ci jobs are using this. (i.e., we can switch off docker and leave the containerd on)

Reply:
Thanks for clarifying. That's what I was wondering if docker and containerd might step on each other. Sounds like not.